benchmarks/fps.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
```

Comment:
Why are we including this file inside the repository? This looks quite specific to the paper, and I don't see many use cases for this script outside of the paper/further FPS optimization.